### PR TITLE
`Development`: Speedup client tests for grading system

### PR DESCRIPTION
**Pull request by ole-ve**
#### General
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.ase.in.tum.de/dev/guidelines/client/).

### Motivation and Context
This PR speeds up the client tests in grading-system.component.spec.ts.

### Description
I replaced some services with the mocked ones, removed unneeded imports and cleaned up beforeEach.

#### Code Review
- [ ] Review 1
- [ ] Review 2

### CODE REVIEW

File: src/test/javascript/spec/component/grading-system/grading-system.component.spec.ts [hunk 1]
```diff
@@ -74,7 +74,7 @@ describe('Grading System Component', () => {
 
     beforeEach(() => {
         TestBed.configureTestingModule({
-            imports: [ArtemisTestModule, TranslateTestingModule, MockModule(FormsModule), RouterTestingModule.withRoutes([])],
+            imports: [ArtemisTestModule, MockModule(FormsModule)],
```
Comment (b-fein) [inline src/test/javascript/spec/component/grading-system/grading-system.component.spec.ts:77]: I think you can remove the import of the `FormsModule` when you add
```
MockDirective(NgModel),
MockDirective(NgSelectOption),
```
to the declarations.

**Review by b-fein — DISMISSED** [review]
I think one module import can be replaced, looks good otherwise. :+1:

File: src/test/javascript/spec/component/grading-system/grading-system.component.spec.ts (hunk 1 above)
Comment (ole-ve) [reply src/test/javascript/spec/component/grading-system/grading-system.component.spec.ts]: Agree, resolved in 10f9bcd.

**Review by b-fein — APPROVED** [review]

**Review by MDK5 — APPROVED** [review]
Code changes look good to me :+1:

**Pull request merged**